### PR TITLE
owners files for pilot packages

### PR DESCRIPTION
**Pull request by rshriram**
This is strictly based on the people who wrote/own major portions of the
code in the respective packages. Let us keep this list small and not make
it a kitchen sink :).

Signed-off-by: Shriram Rajagopalan <shriramr@vmware.com>

### CODE REVIEW

**Comment by rshriram** [comment]
fyi @diemtvu / @yangminzhu for the authn/authz..

File: pilot/pkg/networking/OWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+approvers:
+  - costinm
+  - GregHanson
+  - ijsnellf
+  - rshriram
```
Comment (kyessenov) [inline pilot/pkg/networking/OWNERS]: top-level owners already apply, no need to repeat them.

Comment (kyessenov) [reply pilot/pkg/networking/OWNERS]: thank you for taking over the RDS part :)

**Pull request merged**